src/Adapters/EventSource.Net46/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.ApplicationInsights" version="2.2.0" targetFramework="net45" requireReinstallation="true" />
```

Comment:
Must have been copied from another project inadvertently. Removed. Also the targetFramework updated to 4.6.